src/app/config.rs

```diff
@@ -582,8 +590,10 @@ impl Strings for Env {
 
 impl TestEnv {
     pub fn new() -> Self {
+        let mut values = HashMap::new();
+        values.insert(ENV_PROXY_ID, "foo.deployment.default.linkerd-managed.linkerd.svc.cluster.local".into());
```

Comment:
i'd rather not hardcode this string structure into the proxy, even tests, if possible. The structure will change (soon) and we'll be left with vestigial names. Can we just use an obviously example name like "id.test.example.com" or something?   Or can we just omit it altogether?